content/introduction-and-setup/walkthrough/windows-install/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Windows Users: Visual Studio Community Edition"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 2
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Before you start the installation guide, read through each step below. You will find the recommended features and functionality to install listed in each step.
+
+{{% notice red "Warning" "rocket" %}}
+Allow at least an hour for the installation process.
```

Comment:
Please match the warning from mac-install add emphasis on **an hour** .

Reply:
Done!